st2reactor/st2reactor/rules/filter.py

```diff
@@ -137,7 +137,39 @@ def _render_criteria_pattern(self, criteria_pattern):
         return criteria_pattern
 
 
-class PayloadLookup():
+class SecondPassRuleFilter(RuleFilter):
```

Comment:
Nope. I expect that there will be more rules that fall into the 2nd pass set of rules so trying not to keep it sepcific.